website/docs/d/shared_image.html.markdown

```diff
@@ -43,7 +43,7 @@ The following attributes are exported:
 
 * `location` - The supported Azure location where the Shared Image Gallery exists.
 
-* `generalized` - Is the Operating System present in this Shared Image generalized or not. `false` indicates this image is specialized which for windows images means they have been sysprepped.
+* `specialized` - Is the Operating System present in this Shared Image specialized or not. `true` indicates this image is specialized which for windows images means they have been sysprepped.
```

Comment:
(same here)

Reply:
I understand. I will soon make another PR to update this doc entry